src/base.css

```diff
@@ -1177,21 +1171,37 @@ Possible extra rowspan handling
 
 	.toc a {
 		/* More spacing; use padding to make it part of the click target. */
-		padding-top: 0.1rem;
+		padding: 0.1rem 1px 0;
 		/* Larger, more consistently-sized click target */
 		display: block;
+		/* Switch to using border-bottom */
+		text-decoration: none;
+		border-bottom: 1px solid;
+		border-bottom: 1px solid;
 		/* Reverse color scheme */
 		color: black;
 		color: var(--toclink-text);
+		text-decoration-color: #3980b5;
```

Comment:
Since you've turned off text-decoration on the element, I'm not sure what this is doing.